tools/mmp/driver.cs

```diff
@@ -1191,7 +1191,12 @@ static int Compile ()
 
 				RunCommand (pkg_config, "--cflags mono-2", env, cflagsb);
 				RunCommand (pkg_config, "--variable=libdir mono-2", env, libdirb);
-				RunCommand (pkg_config, "--modversion mono-2", env, mono_version);
+				var versionFile = "/Library/Frameworks/Mono.framework/Versions/Current/VERSION";
```

Comment:
What if mmp is being run via:
`/path/to/custom/mono.exe /Library/Frameworks/Xamarin.Mac.framework/Versions/Current/lib/mmp.exe`?

Reply:
That's a very uncommon scenario (as far as I know nobody's ever done it), and I'm fine with not supporting it (and there are hacks that can be done locally to get around any problems too).